src/status_im/network/net_info.cljs

```diff
@@ -39,6 +40,8 @@
                "type" type
                "details" details)
     (fx/merge cofx
+              (when connectivity-status
```

Comment:
but im not sure you need it if `connectivity-status` hasn't changed, most likely you need to move it to `change-network-status`

Reply:
Don't we turn off Wi-Fi and that makes connectivity status `:offline`

Reply:
yes, but i mean, `connectivity-status` might be the same, so you don't want to send event when it was `:offline` and still `:offline` you want to dispatch only if it has been changed

Reply:
Updated